frontend/src/packages/auth/auth-api.ts

```diff
@@ -77,6 +77,14 @@ class AuthApi extends HttpApi {
       CustomerSignUpResponseDto | BusinessSignUpResponseDto
     >();
   }
+
+  public async logOut(): Promise<unknown> {
+    return await this.load(this.getFullEndpoint(AuthApiPath.LOGOUT, {}), {
+      method: 'GET',
+      contentType: ContentType.JSON,
+      hasAuth: true,
```

Comment:
will it work if token expired?

Reply:
I have connected a JWT validation strategy on endpoint, and it checks for token relevance. With an expired one, the user will not be able to logout.